modules/launcher/src/main/resources/broker.sh

```diff
@@ -120,4 +120,5 @@ $JAVACMD \
     -Dlog4j.configuration="file:$MESSAGE_BROKER_HOME/conf/log4j.properties" \
     -Dbroker.config="$MESSAGE_BROKER_HOME/conf/broker.yaml" \
     -Dbroker.users.config="$MESSAGE_BROKER_HOME/conf/security/users.yaml" \
+    -Dtransports.netty.conf="$MESSAGE_BROKER_HOME/conf/admin-service-transports.yaml" \
```

Comment:
Needs to be introduced in broker.bat too?

Reply:
Done